core/src/main/java/org/javarosa/core/io/StreamsUtil.java

```diff
@@ -7,10 +7,18 @@
 
 public class StreamsUtil {
 
-    private StreamsUtil() {
-        // private constructor
+    public static byte[] getStreamAsBytes(InputStream is) throws IOException {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+        try {
+            StreamsUtil.writeFromInputToOutput(is, baos);
+            return baos.toByteArray();
+        } finally {
+            baos.close();
```

Comment:
minor, but same comment as in the other thread: do we know whether we think it's best practice to not swallow this here? We swallow closes in other contexts, but this might be a better way to do it.